package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@outreach/extensibility-sdk",
   "license": "MIT",
-  "version": "0.6.105",
+  "version": "0.6.107",
```

Comment:
Could we create a new minor version, when adding a new feature?
```suggestion
  "version": "0.7.0",
```

Reply:
Done.